kiali-server/templates/_helpers.tpl

```diff
@@ -16,6 +16,17 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Determine if on OpenShift (when debugging the chart for OpenShift use-cases, set "simulateOpenShift")
+*/}}
+{{- define "kiali-server.isOpenShift" -}}
+{{- if .Values.simulateOpenShift -}}
+true
+{{- else }}
+{{- .Capabilities.APIVersions.Has "operator.openshift.io/v1" -}}
+{{- end -}}
+{{- end }}
```

Comment:
Instead of having another `simulateOpenShift` variable, we could just make `isOpenShift` (or maybe something like `platform: openshift`?) a variable and we only do the `.Capabilities.APIVersions.Has "operator.openshift.io/v1"` check if it's not defined?

Reply:
See latest commit.

There really isn't any other option for "platform" .. it's either OpenShift or it's not. So I just made it look for "isOpenShift" and only do the .Capabilities check if its not defined.

I updated the test procedures in earlier comments to now use "isOpenShift=true" and "isOpenShift=false"... you can omit the `isOpenShift=false` since this will work without having to tell "helm template" that (because the .Capabilities will always return false anyway with "helm template").